companies/fidelity-de.json

```
@@ -0,0 +1,21 @@
+{
+    "slug": "fidelity-de",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "finance"
+    ],
+    "name": "FIL Fondsbank GmbH",
+    "runs": [
+        "Fidelity"
+    ],
+    "address": "Datenschutzbeauftragter\nPostfach 11 06 63\n60041 Frankfurt am Main\nDeutschland",
```

```suggestion
    "address": "Postfach 11 06 63\n60041 Frankfurt am Main\nDeutschland",
```
We add this automatically. :D